src/tests/components/__snapshots__/LoginForm.test.js.snap

```diff
@@ -5,6 +5,7 @@ ShallowWrapper {
   Symbol(enzyme.__root__): [Circular],
```

Comment:
@kakaemma , the snapshot files can be added to your `.gitignore` using `**/__snapshots__/**` unless you need them on origin.

Reply:
to the jest documentationThank you @MuhanguziDavid. However, according to the jest documentation, https://jestjs.io/docs/en/snapshot-testing#should-snapshot-files-be-committed, snapshots should be  committed. That is why I  committed